app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value, number_of_parts = 6, 2
```

Comment:
According to the checklist of README.md - Don't create unnecessary variables for simple tests. Please check this checklist.